decimal64.go

```diff
@@ -474,3 +498,38 @@ func propagateNan(d ...*decParts) *Decimal64 {
 	}
 	return nil
 }
+
+// Class is a miscellaneous operation that takes one operand and provides the class the decimal is in.
+// This function is formally documented here http://speleotrove.com/decimal/damisc.html#refclass.
+func Class(d Decimal64) string {
+	dp := d.getParts()
+
+	switch dp.fl {
+	case flSNaN:
+		return "sNaN"
+	case flQNaN:
+		return "NaN"
+	}
+
+	res := ""
+	if dp.sign == 1 {
+		res += "-"
+	} else {
+		res += "+"
+	}
+
+	switch dp.fl {
+	case flInf:
+		res += "Infinity"
+	case flNormal:
+		if dp.significand.lo == 0 && dp.significand.hi == 0 {
```

Comment:
too specific; If something else in the code changes this will break; We want code that's independent of other code

Reply:
```dp.isZero()```